File: Iniciante/URI 1005.cs
using System;

class URI {

    static void Main(string[] args) {
        double a = double.Parse(Console.ReadLine());
        double b = double.Parse(Console.ReadLine());
        double media = ((3.5 * a) + (7.5 * b)) / 11;
        Console.WriteLine("MEDIA = {0:F5}", media);
    }
    
}

File: Iniciante/URI 1005.dart
import 'dart:io';

void main() {
  double a = double.parse(stdin.readLineSync());
  double b = double.parse(stdin.readLineSync());
  double media = ((3.5 * a) + (7.5 * b)) / 11;
  print("MEDIA = ${media.toStringAsFixed(5)}");
}